mesonbuild/modules/gnome.py

```diff
@@ -2068,7 +2068,7 @@ def _generate_deps(self, state: 'ModuleState', library: str, packages: T.List[st
                 ofile.write(package + '\n')
         return build.Data([mesonlib.File(True, outdir, fname)], install_dir, install_dir, mesonlib.FileMode(), state.subproject)
 
-    def _get_vapi_link_with(self, target: build.CustomTarget) -> T.List[build.LibTypes]:
+    def _get_vapi_link_with(self, target: CustomTarget) -> T.List[build.LibTypes]:
```

Comment:
Seems like you could remove the build from `build.LibTypes` here too.

Reply:
Technically yes, but also not really. It could be done if that also gets imported at the top level, which it currently is.

What I did so far is remove `build.` from any objects that were already imported *anyway*, since in all those cases we might as well use it now that we went to all the effort of importing it.